drivers/docker/fingerprint.go

```diff
@@ -88,5 +89,20 @@ func (d *Driver) buildFingerprint() *drivers.Fingerprint {
 		}
 	}
 
+	if dockerInfo, err := client.Info(); err != nil {
+		d.logger.Warn("failed to get docker system info", "error", err)
+	} else {
+		runtimeNames := make([]string, 0, len(dockerInfo.Runtimes))
+		for name := range dockerInfo.Runtimes {
+			if d.config.GPURuntimeName == name {
+				d.gpuRuntime = true
+			}
+			runtimeNames = append(runtimeNames, name)
+		}
+
+		fp.Attributes["runtimes"] = pstructs.NewStringAttribute(
```

Comment:
technically, "runtimes" attribute is not used, and can be deleted.
Should we keep it?

Reply:
I would keep it. Could be useful for constraints